src/managers/GuildManager.js

```diff
@@ -211,6 +211,25 @@ class GuildManager extends BaseManager {
         }, reject),
     );
   }
+
+  /**
+   * Obtains a guild from Discord, or the guild cache if it's already available.
+   * @param {Snowflake} id ID of the guild
+   * @param {boolean} [cache=true] Whether to cache the new guild object if it isn't already
+   * @returns {Promise<Guild>}
+   * @example
+   * // Fetch a guild by its id
+   * client.guilds.fetch('222078108977594368')
+   *   .then(guild => console.log(guild.name))
+   *   .catch(console.error);
+   */
+  async fetch(id, cache = true) {
```

Comment:
This doesn't allow us to specify `with_counts` per https://discord.com/developers/docs/resources/guild#get-guild-query-string-params

Do we want to just default it to true?

Reply:
Sounds good! I additionally added the force param from #4592